src/chrome/content/rules/Taobao.com-falsemixed.xml

```diff
@@ -1,15 +1,9 @@
 <!--
-	For rules not causing false/broken or valid MCB, see Taobao.xml.
+	For rules not causing false/broken or valid MCB, see Taobao.com.xml.
 -->
-
 <ruleset name="Taobao.com (MCB)" platform="mixedcontent">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="support.taobao.com" />
```

Comment:
The rule is in Taobao_CDN.com.xml
What should I do? Add comments or enable both?